quickwit/quickwit-proto/src/lib.rs

```diff
@@ -21,144 +21,38 @@
 #![deny(clippy::disallowed_methods)]
 #![allow(rustdoc::invalid_html_tags)]
 
-use anyhow::anyhow;
-use ulid::Ulid;
-pub use sort_by_value::SortValue;
 use std::cmp::Ordering;
 use std::convert::Infallible;
 use std::fmt;
+
 use ::opentelemetry::global;
-use ::opentelemetry::propagation::Extractor;
-use ::opentelemetry::propagation::Injector;
+use ::opentelemetry::propagation::{Extractor, Injector};
 use tonic::codegen::http;
 use tonic::service::Interceptor;
 use tonic::Status;
 use tracing::Span;
 use tracing_opentelemetry::OpenTelemetrySpanExt;
+use ulid::Ulid;
 
-pub use tonic;
-
-#[path = "quickwit/quickwit.indexing.rs"]
+pub mod control_plane;
 pub mod indexing;
-#[path = "quickwit/quickwit.metastore.rs"]
+#[path = "codegen/quickwit/quickwit.metastore.rs"]
```

Comment:
At this point, we want some code formatting in `quickwit_proto`, so I moved all the generated files to a `codegen` folder.